### ISSUE TITLE
[Mage] [Spell] "Frost warding" + "Incanter's Absorption" interaction

### ISSUE THREAD

**Issue by Whoopsiexd** [SunwellTracker/issues#2950]
Decription: 
Frost Warding gives your Frost Ward or Fire Ward a 30% chance to negate a spell.
Incanter's Absorption increases your spell damage by 15% of the damage absorbed by Mana Shield, Frost Ward or Fire Ward.

How it works: 
When Frost Warding negates a spell it interacts with Incanter's Absorption and gives you 15% of the damage you negated --> example: if you negate Mimiron's Rocket Strike you get a million sp

How it should work: 
Negation does not equal absorption

Source (you should point out proofs of your report, please give us some source):

**Comment by Vaxsysl** [comment]
Yeah this is a weird one. And yes, it many ways it shouldn't. Also hi, I'm the author of "[that](https://sunwell-community.com/index.php?/topic/1032-mage-pve-guide-arcane-fire-raiding-best-in-slot-lists-t07-t10/)" guide on sunwell forums.

Here is the thing. I was writing my guide back in 2012 first, then scratched it and remade it, twice. During the third re-write in 2014 I was pruning through official forums thread by thread each day and digging through logs. From what I could understand it, the first nerf to the talent was after mages could solo clear Military wing on their own. Then it received another nerf so only mage's own absorbs could increase spellpower.

I was also on retail in 2009 and I rarely saw Incanter's Absorption teamed up with Frost Warding. Rarely any player takes either talent, even FFB mages (??), focusing instead on their raw potential on all bosses and all encounters instead of having a quirk at some others.

So it bugged me a lot. Saying that having both with some risk of dying can be potent would be oversimplifying it (or be alive with Holy Priest in, then pray to GCD god or get revived and do it again). You stepped one one of the most OP scenarios and there are two in whole expansion, both on the same fight. It's not game-breaking in/on any other (minus enrage dragon breaths god I love those!).

Anyway, so I pruned the official forums six-seven years ago. After months of research for the guide I could find nothing about them NOT supposing to work together in 3.3.5a. Ulduar was behind and there's no game-breaking "use" of this afterwards, no frost or fire nuke is so strong in the game anymore for the mages to make use of this in such OP way. 

Is it OP for this specific thing? Yes it is. Script logic with negated being absorbed is on the sideline, but really, I found no post or word saying it shouldn't work together. Only the opposite. If you find proof... post it. Official forums from WotLK era are mostly scrapped and unavailable these days.

**Comment by Vaxsysl** [comment]
That being said, if staff wants to "sort this out", I suggest - for the time being - that said rocket and water explosion on hardmode are treated as unresistable.

Really guys, I was digging and browsing and I found nothing to say "Nope!" to.

**Comment by Whoopsiexd** [comment]
It's true. I've also looked into how exactly it worked and  all the sources I was able to find say that the negated damage does count towards Incanter's Absorption, which means it should work exactly how it works now. 
https://criticalqq.wordpress.com/2009/05/14/incanters-absorption/
The only fight that this really is OP in is in Mimiron and since this talent combination was available only many patches after the release of Ulduar it seems that during those times this passed on as unnoticed due to Mimiron being old content that pretty much nobody needed upgrades from.
